packages/data-store/src/migrations/3.createPrivateKeyStorage.ts

```diff
@@ -54,12 +52,14 @@ export class CreatePrivateKeyStorage1629293428674 implements MigrationInterface
     await queryRunner.manager
       .createQueryBuilder()
       .insert()
-      .into(getTableName('private-key'))
+      .into(getTable('private-key').name)
       .values(privKeys)
       .execute()
     // 3. drop old column
     debug(`dropping privKeyHex column from old key table`)
-    await queryRunner.dropColumn(getTableName('key'), 'privateKeyHex')
+    if (getTable('key').columns.filter((column => column.name === 'privateKeyHex')).length > 0) {
```

Comment:
Is there a need to add the expression? The original also didn't have it and it is highly likely that dropColumn is idempotent

Reply:
The column does not exist when the query is run. It's not there each time. This was precisely the reason why in my previous PR there were some changes in the entity. Basically in this one as well there is this line that is not going to work. because the column is not there in the first place.

Reply:
That suggests the code is not resulting in consistent results, which I find highly unlikely tbh.